kubejs/client_scripts/item_modifiers/jei_descriptions.js

```diff
@@ -491,9 +491,9 @@ onEvent('jei.information', (event) => {
 
     recipes.forEach((recipe) => {
         for (let i = 0; i < recipe.text.length; i++) {
-            if (recipe.with[i]) {
+            try {
```

Comment:
How about doing something like this:
```js
if (Object.keys(recipe).includes("with")) {
    recipe.text[i] = Text.translate(recipe.text[i], recipe.with[i]);
} else {
    recipe.text[i] = Text.translate(recipe.text[i]);
}
```

Reply:
The main issue is that the recipe.text and recipe.with arrays can be different lengths, checking whether recipe.with exists is simply `if (recipe.with){...}`

Reply:
exactly 👍